cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/urfave/cli/v2"
 
 	"github.com/NVIDIA/nvidia-container-toolkit/internal/logger"
+	pkgconfig "github.com/NVIDIA/nvidia-container-toolkit/pkg/config"
```

Comment:
nit: If we rename the `config` struct `options` instead, we don't need to rename the import.

Reply:
Agreed that `pkgconfig` is non-specific. I am trying to think of a better name for the package that will hold the `Loader` interface and its implementations